.github/workflows/tx-pull.yml

```diff
@@ -9,13 +9,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
https://github.com/stefanzweifel/git-auto-commit-action#commits-made-by-this-action-do-not-trigger-new-workflow-runs

☝🏽 implies passing the token here would allow the action to inherit the write authority to `main` (unless it's protected in another way I did not account for)

Reply:
I think github already uses the token and we only need to pass token if we need to use a specific PAT

Reply:
Token will be needed to create commits on the protected branch: https://github.com/ipfs/ipfs-webui/pull/2053/files#diff-62bcbd77df88ba2eaae2e825b94fe6e1cdbbfb09690a17b094c9a628a2d64b31L78